### PR TITLE
updates to subscribe widget & modal sub text

### PR DESCRIPTION
**Pull request by djnunez-aot**
#680 
https://github.com/bcgov/met-public/issues/#680

-Change Sign up to Sign Up

-Make subscribe modal subtext bold:
  To do this we added the subtext prop to all modal subtexts for future use cases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by tiagograf-AOT — APPROVED** [review]

File: met-web/src/components/common/Modals/types.tsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
+export interface ModalSubtext {
+    text: string;
+    bold: boolean;
```
Comment (jadmsaadaot) [inline met-web/src/components/common/Modals/types.tsx:3]: bold should be an optional field bold?: string. And we don't have to do bold: false when we create a ModalSubtext, we only pass it if it's true

File: met-web/src/components/engagement/form/AddSurveyBlock.tsx [hunk 2]
```diff
@@ -75,8 +75,14 @@ export const AddSurveyBlock = ({
                 data: {
                     header: 'Remove Survey',
                     subText: [
-                        'You will be removing this survey from the engagement. This survey will not be deleted and will be available to add to any engagement.',
-                        'Do you want to remove this survey?',
+                        {
+                            text: 'You will be removing this survey from the engagement. This survey will not be deleted and will be available to add to any engagement.',
+                            bold: false,
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/form/AddSurveyBlock.tsx:80]: we shouldn't pass it if it's false

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
false parameter should be an optional one

File: met-web/src/components/engagement/form/AddSurveyBlock.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/form/AddSurveyBlock.tsx]: changed

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**